src/commands.js

```diff
@@ -173,7 +175,243 @@ module.exports = {
       }
       res.end()
     }
+  },
+  hide: {
```

Comment:
I like the concept of 'hide' rather than 'ban' since the user can still connect to peers and see other people's IP addresses & messages!

Other applications call this 'mute' -- can we use that instead? I think it'll be easier to grok for new users, but happy to explore 'hide' as well if you're tied to it or have user research that backs up using 'hide' rather than 'mute' ... 

If there is single-user mute as well, could add 'mute just for me' vs. 'mute for the whole cabal' 

in the future, a `ban` command could be effectively adding that peer's ip to a blocklist at the network layer, which is a somewhat effective tactic used in irc to prevent DDOS and spam.

Reply:
@okdistribute we intend to have mute as well. the difference is that `/mute` will preserve all previous messages while not showing any future ones, while `/hide` will hide both previous and future messages.

does that make sense?

`mute just for me` is a really nice way of phrasing private mutes/hides. we've been talking about that all moderation actions should be able to be taken privately as well, by which we would encrypt their contents

Reply:
It's weird (at least in my brain): for private (personal) moderation, I'd expect MUTE to work like it does on Twitter & Mastodon, and hide their past/present/future content. However, as a public moderation operation, I'd expect MUTE to hide just future content. There's something about the fact that the moderation operation may be adopted by other users that makes me not expect the user to suddenly be "vanished" from historic records.

Reply:
I agree, this distinction between mute & hide seems to be very nuanced and if *we're* confused about it already, we could expect new users to be 10x more confused xD

Reply:
+1. We're going to keep thinking about this.